src/contrib/mlir/compiler/pass/mlir_subgraph_extraction.cpp

```diff
@@ -39,6 +39,7 @@
 #include "ngraph/op/negative.hpp"
 #include "ngraph/op/relu.hpp"
 #include "ngraph/op/subtract.hpp"
+#include "ngraph/pattern/op/label.hpp"
```

Comment:
I rather not abuse label nodes and maybe add a different node for that purpose. TensorIterator should have the same issue. Let's discuss it more.